scripts/updates/update_phot_filters.py

```diff
@@ -0,0 +1,80 @@
+from sqlalchemy import func, and_
+from astrodb_utils import load_astrodb
+from simple.schema import *
+from simple.schema import REFERENCE_TABLES
+import pandas as pd
+
+# Establish connection to database
+db = load_astrodb("SIMPLE.sqlite", recreatedb=True, reference_tables=REFERENCE_TABLES)
+
+# Run a query to find the bands with null UCDs
+t = db.query(
+    db.PhotometryFilters.c.band,
+    db.PhotometryFilters.c.ucd,
+    db.PhotometryFilters.c.effective_wavelength,
+).pandas()
+null_values = pd.isnull(t["ucd"])
+df = t[null_values].sort_values(by=["effective_wavelength"])
+df_new = df.drop(columns=["effective_wavelength"])
+data_dicts = df_new.to_dict(orient="records")
```

Comment:
While this is fine, you may have been able to do the same with:
```
t = db.query(
    db.PhotometryFilters.c.band,
    db.PhotometryFilters.c.ucd,
    db.PhotometryFilters.c.effective_wavelength,
)
.filter(db.PhotometryFilters.c.ucd.is_(None))
.order_by(db.PhotometryFilters.c.effective_wavelength)
.pandas()
```

Reply:
I couldn't figure out the NONE!!!  Is that construction documented somewhere?  In SQLAlchemey docs or someplace else?

Reply:
It's likely in the SQLAlchemy docs, but those can be hard to navigate sometimes. I ended up looking at I think the integrity tests since we had done similar queries before.